File: Examples/Node.js/Document/ReplaceDocTextList.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var PdfApi = require('asposepdfcloud');
var StorageApi = require('asposestoragecloud');
var configProps = require('../config/config.json');
var data_path = '../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var outFolder = configProps.out_folder;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose.Pdf API SDK
var pdfApi = new PdfApi(config);

var name = "Sample-Annotation.pdf";	
var tr1 = {
			'OldValue' : 'Sample',
			'NewValue' : 'Sample Aspose'			
	};
	
var tr2 = {
			'OldValue' : 'PDF',
			'NewValue' : 'Aspose PDF'			
	};
	
var textReplaceListRequestBody = {
			'TextReplaces' : [tr1, tr2]
			
	};

try {
// Invoke Aspose.Storage Cloud SDK API to upload file
storageApi.PutCreate(name, null, null, data_path + name , function(responseMessage) {
		assert.equal(responseMessage.status, 'OK');

		// Invoke Aspose.Pdf Cloud SDK API to replace pdf text list
		pdfApi.PostDocumentReplaceTextList(name, null, null, textReplaceListRequestBody, function(responseMessage) {
			console.log("Status:" + responseMessage.status);
			
			});
	});

}catch (e) 
{
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1

File: Examples/DotNET/CSharp/Document/ReplaceDocTextList.cs
using System;
using Com.Aspose.PDF.Api;
using Com.Aspose.PDF.Model;
using Com.Aspose.Storage.Api;

namespace Document
{
    class ReplaceDocTextList
    {
        public static void Run()
        {
            // ExStart:1
            PdfApi pdfApi = new PdfApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            string fileName = "test.pdf";
            string storage = null;
            string folder = null;

            Com.Aspose.PDF.Model.TextReplaceListRequest body = new Com.Aspose.PDF.Model.TextReplaceListRequest();
            Com.Aspose.PDF.Model.TextReplace tr = new Com.Aspose.PDF.Model.TextReplace();
            tr.NewValue = "This will be the new text";
            body.TextReplaces = new System.Collections.Generic.List<Com.Aspose.PDF.Model.TextReplace> { tr };

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate( fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.PDF Cloud SDK API to replace pdf text list
                DocumentTextReplaceResponse apiResponse = pdfApi.PostDocumentReplaceTextList(fileName, storage, folder, body);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    Console.WriteLine("Replace PDF Document Text List, Done!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
